.github/actions/build-push-image/action.yaml

```diff
@@ -88,34 +82,65 @@ runs:
       shell: bash
       run: |
         go mod download
-    - name: Build and Push Operator to Docker Registry
+    - name: Build all platforms
+      shell: bash
+      run: make all-platforms
+    - name: Fail if send input flags are both unset
+      if: ${{ inputs.push_to_docker != 'true' && inputs.push_to_quay != 'true' }}
+      shell: bash
+      run: exit 1
+    - name: Compute main build tag
```

Comment:
I don't get why you added this step if when pushing you're prefixing quay.io anyway

Reply:
No, this tag job decides if we are pushing to docker OR quay in the build and push, depending on the input flags. See the table of truth in the other comment.